test/missiles_test.cpp

```diff
@@ -4,6 +4,15 @@
 
 using namespace devilution;
 
+TEST(Missiles, RotateBlockedMissileArrow)
+{
+	Player &player = Players[0];
+	auto *missile = AddMissile({ 0, 0 }, { 0, 0 }, Direction::South, MIS_ARROW, TARGET_MONSTERS, player.getId(), 0, 0);
+	EXPECT_EQ(missile->_miAnimFrame, 1);
+	TestRotateBlockedMissile(*missile);

```

Comment:
This relies on an RNG call, you'll need to seed the engine before calling TestRotateBlockedMissile to make it consistent. See drlg_l*_test, quests_test, or random_test for examples.

Reply:
Thanks!